packages/uiweb/src/lib/hooks/useResolveWeb3Name.ts

```diff
@@ -2,51 +2,38 @@
 import type { Env } from '@pushprotocol/restapi';
 import { ethers } from 'ethers';
 import { useContext, useEffect, useState } from 'react';
-import { CoreContractChainId, ENV, InfuraAPIKey } from '../config';
 import { ChatMainStateContext } from '../context';
 import type { ChatMainStateContextType } from '../context/chatAndNotification/chat/chatMainStateContext';
 
 // Internal Components
 
-
-
 // Internal Configs
-import { getEnsName, getUdResolver, getUnstoppableName, pCAIP10ToWallet } from '../helpers';
-import { Web3NameListType } from '../types';
-
-
-
-export function useResolveWeb3Name(address: string,env:Env) {
-
-  const { web3NameList,setWeb3Name,selectedChatId } = useContext<ChatMainStateContextType>(ChatMainStateContext);
+import { pCAIP10ToWallet, resolveWeb3Name } from '../helpers';
 
+export function useResolveWeb3Name(address: string, env: Env) {
```

Comment:
You said we are not using it now and it's only used in older components. I see it's used in only 2 other components. Can we replace this hook and get rid of it?

Reply:
This can be done in another issue, because in the those components, we will have to replace quite a few things depending on this change